src/index.js

```diff
@@ -102,6 +118,7 @@ async function run(params) {
     return serviceembed(params, url, log);
   }
 
+  await disconnectAll();
```

Comment:
this isn't really good here, as it disconnects all (concurrent) fetchs requests. 
I would create a context for each action invocation - then you can also enforce http1 for tests.